### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by armandocomellas1**
### The mani purpose of this pull request are:

- Have to install both React and Redux packages.
- Initialise components and connect them to each other to interact with the states.
-  Understand the concept of state management.
- Use store, actions, and reducers in React.
- Good practices such as Ducks.

Thank you team!
💯

### CODE REVIEW

File: src/redux/books/books_duck.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+// Actions creators
+export const addBook = (props) => {
+  return {
+    type: ADD,
+    book: props,
+  };
+};
+
+export const delBook = () => {
```
Comment (SajjadAhmad14) [inline src/redux/books/books_duck.js]: The user is not able to delete a book item. Kindly make sure a book item can be removed from a list.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Armando, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/redux/books/books_duck.js (hunk 1 above)
Comment (armandocomellas1) [reply src/redux/books/books_duck.js]: Nice observation. thanks! solved!

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

# Hi @armandocomellas1 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**